dte/src/components/Shared/FormElements/CommonElements/ResendEmail.tsx

```diff
@@ -18,12 +18,12 @@ const ResendEmail = ({ emailAddress }: ResendEmailProps) => {
       url: `${process.env.REACT_APP_BASE_API}/users/resendverificationemail`,
       method: "POST",
       data: {
-        email: emailAddress,
+        email: userId,
```

Comment:
data looks wrong here, in my file it is just userId which is js shorthand for userId: userId so the data will read differently on the api